pynecone/components/component.py

```diff
@@ -480,7 +488,7 @@ def get_custom_components(self) -> Set[CustomComponent]:
         return (
             {self}
             | super().get_custom_components()
-            | self.get_component().get_custom_components()
+            # | self.get_component().get_custom_components()
```

Comment:
I will add this back in a follow-up, it's currently causes an infinite recursion in some cases.